src/System.Buffers.Primitives/System/Buffers/OwnedArray.cs

```diff
@@ -19,7 +20,7 @@ public OwnedArray(int length)
 
         public OwnedArray(T[] array)
         {
-            if (array == null) BufferPrimitivesThrowHelper.ThrowArgumentNullException(nameof(array)); 
+            if (array == null) new ArgumentNullException(nameof(array)); 
```

Comment:
nit: use throw helper here like in other places

Reply:
will do